Shopping List/Shopping List/Controller/Table with shopping items/ItemTableViewCell.swift

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
Why it is in `Controller` group?

Reply:
My cell doesn't work with the model, so I will move it to the `View` group.

Reply:
Done.